nvflare/fuel/f3/stream_cell.py

```diff
@@ -30,8 +27,6 @@ def __init__(self, cell: CoreCell):
         self.byte_streamer = ByteStreamer(cell)
         self.byte_receiver = ByteReceiver(cell)
         self.blob_streamer = BlobStreamer(self.byte_streamer, self.byte_receiver)
```

Comment:
just to confirm, we don't need "ObjectStreamer" and "FileStreamer" anymore?
now we always use ByteStreamer and BlobStreamer

Reply:
Correct. Those two are implemented at higher level so they are duplicate.